src/main/java/com/openshift/internal/client/ApplicationResource.java

```diff
@@ -555,14 +555,13 @@ public IGearGroup getGearGroup(ICartridge cartridge) throws OpenShiftException {
 	}
 
 	public IGearGroup getGearGroup(ICartridge cartridge, Collection<IGearGroup> gearGroups) {
-		if (cartridge == null
-				|| gearGroups == null) {
+		if (cartridge == null || gearGroups == null) {
```

Comment:
I personally prefer to have each condition in a separate line for better readability. Not a reason to refuse this PR though.